code/frontend/MakeItSo/MakeItSo/MakeItSo.docc/Tutorials/Building a simple todo list UI.tutorial

```diff
@@ -0,0 +1,473 @@
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-02-contentview-05.swift) {
+          @Image(source: building-todo-ui-reminder-02-contentview-04.png, alt: "")
+        }
+      }
+      
+      @Step {
+        Wrap the `HStack` in a `List`. 
+        
+        > Warning: This will result in a compile error saying that `Reminder` is not `Identifiable`. You will fix this in the next step.
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-02-contentview-06.swift) {
+          @Image(source: building-todo-ui-reminder-02-contentview-06.png, alt: "")
+        }
+      }
+      
+      @Step {
+        Mark `Reminder` as `Identifiable`, and add a constant property `id`.
+        
+        > Note: SwiftUI will use the `id` property of the `Reminder` struct to determine the identity of the list rows inside the `List` view. It is important to keep a view's identity stable, as this will help SwiftUI determine which parts of the UI to update when the data model changes.
+        
+        @Code(name: "Reminder.swift", file: building-todo-ui-reminder-02-reminder-02.swift, previousFile: building-todo-ui-reminder-02-reminder-01.swift)
+      }
+      
+      @Step {
+        Navigate back to `ContentView.swift`. The compile error should now be resolved, and the error message will disappear. 
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-02-contentview-07.swift) {
+          @Image(source: building-todo-ui-reminder-02-contentview-07.png, alt: "")
+        }
+      }
+      
+      The `List` view iterates over all elements of the collection you pass in as the first parameter - in this case, `reminders`. Inside the `List` view's closure, you can access the individual elements of the collection using the closure parameter, `reminder`.
+      
+      @Step {
+        Update the views inside the `List` view to display the `title` and `completion` state of the current collection element.  
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-02-contentview-08.swift) {
+          @Image(source: building-todo-ui-reminder-02-contentview-08.png, alt: "")
+        }
+      }
+      
+    }
+  }
+  
+  @Section(title: "Mark Reminders as Completed") {
+    @ContentAndMedia {
+      The main purpose of a todo list app is to allow users to get things done. In this step, you will turn the static `Image` view into an interactive component that allows users to mark tasks as `completed`.
+      
+      @Image(source: placeholder-image.png, alt: "")
+    }
+    
+    @Steps {
+      @Step {
+        In the _Project_ navigator, select `ContentView.swift`.
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-03-contentview-01.swift) {
```

Comment:
Just a nit - the ContentView doesn't get highlighted on this step unlike on the step on line 74.

Reply:
Should be fixed now.